.github/workflows/release.yml

```diff
@@ -13,7 +13,7 @@ jobs:
         with:
           fetch-depth: 0
       - uses: ./.github/actions/prepare
-      - run: pnpm build:release
+      - run: pnpm build
```

Comment:
The release ci job was never releasing an esm build, only cjs.
This is no fixed.